### PR TITLE
grammar/style editing

### PR DESCRIPTION
**Pull request by Rachael-Graham**
ran through the Kube quickstart & made small grammar & style changes throughout. Some diffs are simply spacing changes to make lines easier to review. @rshriram could you take a look?

### CODE REVIEW

**Comment by Rachael-Graham** [comment]
signed the CLA

File: _docs/setup/install-kubernetes.md [hunk 1]
```diff
@@ -126,39 +130,50 @@ You can find out more about how to use these tools in [Collecting Metrics and Lo
...
 <figure><img src="./img/servicegraph.png" alt="BookInfo Service Graph" title="BookInfo Service Graph" />
 <figcaption>BookInfo Service Graph</figcaption></figure>
 
 
 ## Verifying the installation
 
-1. Ensure the following Kubernetes services were deployed: "istio-pilot", "istio-mixer", "istio-ingress", "istio-egress",
-   "istio-ca", and, optionally, "grafana", "prometheus' and "servicegraph".
+1. Ensure the following Kubernetes services are deployed: `istio-pilot`, `istio-mixer`,
+   `istio-ingress`, `istio-egress`, and, optionally, `grafana`, `prometheus` and `servicegraph`.
```
Comment (Rachael-Graham) [inline _docs/setup/install-kubernetes.md]: `istio-ca` is not shown in the output

Comment (ldemailly) [reply _docs/setup/install-kubernetes.md]: It should be

**Comment by ldemailly** [comment]
Can you attach a screenshot/double check the layout and indentation hasn't accidentally changed ?

File: _docs/setup/install-kubernetes.md (hunk 1 above)
Comment (Rachael-Graham) [reply _docs/setup/install-kubernetes.md]: Hmm okay. Whoever originally wrote this doc didn't get it in their output, and I didn't get it in mine either. perhaps I'm doing something wrong

Comment (rshriram) [reply _docs/setup/install-kubernetes.md]: can you add istio-ca back? This is a bug and we can fix it. Or may be we can fix it later, when someone updates the output.

**Comment by rshriram** [comment]
I don't see any spacing changes for the commands (except one, which is basically aligning things properly).

**Pull request merged**